config/locales/en/claims/support/claims/payments.yml

```diff
@@ -5,4 +5,11 @@ en:
         payments:
           index:
             heading: Claims
-            sub_heading: Payments
+            sub_heading:
+              zero: Payments
+              one: Payments (%{count})
```

Comment:
It'd be nice if we could omit `one` here, is that a possibility or does it fall over?

Reply:
Yeah, it falls over 😢